frontend/src/components/coverage/enrollment/ResidentDetails.vue

```diff
@@ -403,3 +413,8 @@ export default {
   },
 }
 </script>
+<style scoped>
+.text-red {
```

Comment:
Given that this a scoped style you could give this class a more specific name that refers to date of death.